### PR TITLE
Add functionality for ahelp relaying, pm command

### PR DESCRIPTION
**Pull request by MuckerMayhem**
Adds functionality for relaying ahelps from the server to Discord, and adds the /pm command for admins to PM users on the server from Discord.

### CODE REVIEW

**Comment by MuckerMayhem** [comment]
I fully expect the ahelp relaying bit to be done better soon, I just couldn't figure a better way right now while we still have this weird 'multi server' thing going on, but it should be easier once that is gutted.

File: src/servers/baystation.js [hunk 1]
```diff
@@ -53,9 +54,29 @@ export const baystation = new class extends Server {
...
+
 }({
   name: 'Baystation',
-  port: 8000,
-  host: 'baystation.xyz',
-  guild: '104289531796131840'
+  port: 55143,
+  host: '192.168.0.215',
+  guild: '119249275870904322'
 })
```
Comment (SierraKomodo) [inline src/servers/baystation.js:82]: Was this change intended to be pushed?

Comment (MuckerMayhem) [reply src/servers/baystation.js:82]: Nop, will undo
